docs/UserGuide.md

```diff
@@ -46,15 +47,12 @@ Do have a look at our Table of Contents to locate the help you need!
 ## Quick Start <a name="quick_start"></a>
 
 1. Ensure you have `Java 11` or above installed on your Computer.
-   If you don't have `Java 11`, simply click [here](https://www.oracle.com/java/technologies/downloads/) to download the latest Java Development Kit.
-
+   If you already have `Java` installed, you might want to type in `java -verson` in your command window to ensure you are running on the correct version of `Java`!
+   If you don't have `Java 11`, simply click [here](https://www.oracle.com/java/technologies/downloads/), navigate to the "Java SE Development Kit 11.*.**" section to download the latest Java Development Kit.
 2. Download the latest `socialite.jar` from [here](https://github.com/AY2122S1-CS2103T-F11-4/tp/releases).
-
 3. Copy the file to the folder you want to use as the _home folder_ for your SociaLite.
-
-4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. SociaLite should start in a few seconds! The app contains some sample data for you experiment with to get familiar with the commands.<br>
```

Comment:
Missing "to". Change to: "The app contains some sample data for you to experiment with and get familiar with the commands."